### DISCUSSION TITLE
Old issues

### DISCUSSION THREAD

**Discussion by rishujam** [kiwix/kiwix-android#3221] [Q&A] [answered]
1. Are the issues which are old around 2020,2019, etc still need to be implemented if not so why are they still open?
2. There are many issues whose PR has been created and is still open. 
3. Single person is assigned many issues.

**Comment by kelson42** [comment, marked as answer]
> 1. Are the issues which are old around 2020,2019, etc still need to be implemented if not so why are they still open?

Probably yes.

> 2. There are many issues whose PR has been created and is still open. 

Yes, at bit less than 20.

> 3. Single person is assigned many issues. 

Yes, there is currently someone working fulltime

**Comment by rishujam** [comment]
Thanks for the answer
